controllers/grades.go

```diff
@@ -0,0 +1,50 @@
+package controllers
+
+import (
+	"context"
+	"github.com/BetterGR/grades-microservice/protos"
+	"google.golang.org/grpc/credentials/insecure"
+	"log"
+	"net/http"
+	"time"
+
+	"github.com/gin-gonic/gin"
+	"google.golang.org/grpc"
+)
+
+var grpcClient protos.GradesServiceClient
+
+func InitGRPCClient() {
+	// Initialize the gRPC client connection.
+	conn, err := grpc.NewClient("localhost:50051", grpc.WithTransportCredentials(insecure.NewCredentials()))
```

Comment:
It would be better to have the function receive the client address. For now, the caller will pass the string (from consts) - but later service-discovery may assist.